spec/ParseLiveQueryServer.spec.js

```diff
@@ -899,7 +909,74 @@ describe('ParseLiveQueryServer', function () {
       expect(client.pushDelete).not.toHaveBeenCalled();
       expect(client.pushLeave).toHaveBeenCalled();
       done();
-    }, jasmine.ASYNC_TEST_WAIT_TIME);
+    }, ASYNC_TEST_WAIT_TIME);
+  });
+
+  it('can handle object multiple commands which matches some subscriptions', async done => {
```

Comment:
To make this more specific, how about:

> sends correct events for object with multiple subscriptions

Reply:
Sorry, I'm not very good in English when it comes to that kind of things 😂

Reply:
This is not easy to describe for me neither 😄

Reply:
Ah ah you're reassuring me!